src/renderer/Components/CreatingPackage/PackagePreview/Order/index.jsx

```diff
@@ -0,0 +1,46 @@
+import usePackageStore from "../../../../store";
+import PropTypes from "prop-types";
+
+function Order({ order, index }) {
+  const { deleteOrder } = usePackageStore();
+
+  const { action, attachmentName, sourcePath, executionPath, editingName } =
+    order;
+  const handleDelete = () => {
+    deleteOrder(index);
+  };
+
+  return (
+    <div className="item-center container flex justify-items-start">
+      <div className="flex items-center p-2">
+        <div className="py-2 pr-4 text-2xl font-medium text-slate-500">
+          {index + 1}
+        </div>
+      </div>
+      <div className="mx-auto my-2 flex w-fit flex-row items-center justify-start justify-items-start space-x-4 rounded border border-solid border-gray-300 p-2">
+        {sourcePath && (
+          <div className="text-block-blue">{`${sourcePath} 의`}</div>
+        )}
+        <div className="text-block-blue">{`${attachmentName} 을(를)`}</div>
+        <div className="text-block-blue">{`${executionPath} 에`}</div>
+        {editingName && (
+          <div className="text-block-blue">{`${editingName} 으로`}</div>
+        )}
+        <div className="text-block-blue">{`${action}`}</div>
```

Comment:
```suggestion
          <div className="text-block-blue">{sourcePath} 의</div>
        )}
        <div className="text-block-blue">{attachmentName} 을(를)</div>
        <div className="text-block-blue">{executionPath} 에</div>
        {editingName && (
          <div className="text-block-blue">{editingName} 으로</div>
        )}
        <div className="text-block-blue">{action}</div>
```
혹시 이렇게 정리해보는건 어떠신가여? 😄

Reply:
매의 눈이시군요! 좋은 발견 감사합니다. 👍